exporters/ibmmq/exporter.yml

```diff
@@ -1,32 +1,34 @@
 # name of the exporter, should match with the folder name
 name: ibmmq
 # version of the package created
-version: 0.3.0
+version: 0.4.0
 # Relative path to the License path from the repository root
 exporter_license_path: LICENSE
 # URL to the git project hosting the exporter
 exporter_repo_url: https://github.com/ibm-messaging/mq-metric-samples
 # Tag of the exporter to checkout
 exporter_tag:
 # Commit of the exporter to checkout (used if tag property is empty)
-exporter_commit: d268624ff90d2fbbec18d62b6f73ed8955c4ee8f
+exporter_commit: 322e8f6cc5ac687e5766acd4eaee447cc7d62d7a
```

Comment:
You took the chance to update the exporter, right? Nice!

Reply:
yes, we were pointing to an off-tag release and now we are back on track